release-notes/opensearch-release-notes-1.3.3.md

```diff
@@ -0,0 +1,41 @@
+# OpenSearch and OpenSearch Dashboards 1.3.3 Release Notes
+
+## Release Details
+
+OpenSearch Dashboards 1.3.3 includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.3.md).
```

Comment:
Also why we don't have notes for OpenSearch? Probably put the supposed link here if they don't have it ready yet.

Reply:
Hey @zelinh  I dont see any changes to OpenSearch 1.3, hence I have not added it.
https://github.com/opensearch-project/OpenSearch/tree/main/release-notes
https://github.com/opensearch-project/OpenSearch/commits/1.3